### PR TITLE
test: Empty memory support

### PR DESCRIPTION
**Pull request by gumb0**

### CODE REVIEW

File: test/unittests/capi_test.cpp [hunk 1]
```diff
@@ -145,6 +145,18 @@ TEST(capi, has_memory)
 
     fizzy_free_module(module_no_memory);
 
+    /* wat2wasm
+      (memory 0)
+    */
+    const auto wasm_memory_empty = from_hex("0061736d010000000503010000");
```
Comment (axic) [inline test/unittests/capi_test.cpp:151]: Why is this duplicated in two test cases?

Comment (gumb0) [reply test/unittests/capi_test.cpp:151]: One test here checks module inspection function `fizzy_module_has_memory`

The test below checks instance memory access functions.

**Review by axic — APPROVED** [review]

File: include/fizzy/fizzy.h [hunk 2]
```diff
@@ -450,6 +450,8 @@ const FizzyModule* fizzy_get_instance_module(FizzyInstance* instance);
 /// @return             Pointer to memory data or NULL in case instance doesn't have any memory.
 ///
 /// @note    Function returns pointer to memory regardless of whether memory is exported or not.
+/// @note    For instances of the modules defined with memory of size 0 the returned pointers is not
```
Comment (chfast) [inline include/fizzy/fizzy.h:453]:
```suggestion
/// @note    For instances of the modules defined with memory of size 0 the returned pointers are not
```

**Review by chfast — APPROVED** [review]

File: include/fizzy/fizzy.h (hunk 2 above)
Comment (axic) [reply include/fizzy/fizzy.h]: Well, shouldn't it be just `pointer is not`, since it is a single pointer?

**Pull request merged**